internal/generator/vector/output/syslog/rfc3164_with_defaults.toml

```diff
@@ -0,0 +1,43 @@
+[transforms.example_parse_encoding]
+type = "remap"
+inputs = ["application"]
+source = '''
+. = merge(., parse_json!(string!(.message))) ?? .
+if .log_type == "infrastructure" && .log_source == "node" {
```

Comment:
@vparfonov There might be logs with .log_type == "infrastructure" that do not have .log_source == "node". As far as I can see, I found that the logs from namespace_name=openshift-kube-apiserver and container_name=kube-apiserver do not have .log_source == "node". This line may need a conditional statement for container logs within the openshift-* namespace.

Reply:
@kattz-kawa Good point, it can be .log_source == "container", but it will be not journal log something close to application logs.

Reply:
I think this logic is correct, could simplify slightly :
- if (node) - these are journald logs and we can map the syslog fields exactly from the logs.
  Note: 'if (node)' is equivalent to 'if (infra && node)' because all node logs are infra logs.
  The difference in format is really node vs. container, not application vs. infra.
- if (container) - container logs are formatted the same regardless of type (application or infra)
- if (audit) - audit logs formatted differently from either of the previous two cases.